src/lib/Core.jsx

```diff
@@ -287,6 +357,11 @@ const Core = function ({
           .replace('<correctPoints>', correctPoints)
           .replace('<totalPoints>', totalPoints)}
       </h2>
+      <h2>
+        {correctPoints < 40
```

Comment:
the passing mark should be configurable. don't hardcode 40.

Reply:
It was  returned that predefined value for pass or fail but was not given so I took it to be 40

Reply:
in some countries, the passing mark is 50 or even 60. So this needs to be configurable.